common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -2413,7 +2413,8 @@
   },
   "INVALID_FRACTION_OF_SECOND" : {
     "message" : [
-      "The fraction of sec must be zero. Valid range is [0, 60]. If necessary set <ansiConfig> to \"false\" to bypass this error."
+      "The fraction of sec must be zero. Valid range is [0, 60]. To avoid this error, use try_make_timestamp, which returns NULL on error.",
```

Comment:
Note that this is a runtime error, so the user may not know hat the bad value is. I think it would be beneficial if we were able to return the bad value